lib/partial/transmuxer.js

```diff
@@ -51,7 +51,7 @@ var tsPipeline = function(options) {
 
   // Hook up CEA-608/708 caption stream
   pipeline.h264
-   .pipe(pipeline.captionStream);
+    .pipe(pipeline.captionStream);
```

Comment:
Extra space

Reply:
I think this is actually correct though? 2 spaces rather than the 1 space that was there before